### DISCUSSION TITLE
Welcome to Homebridge Honeywell Home Discussions!

### DISCUSSION THREAD

**Discussion by donavanbecker** [d0n4v4nb3ck3r/homebridge-honeywell-home#331] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by honeywelluser** [comment]
Hello,
I’m new to this. I’ve tried the competitor's https://github.com/luc-ass/homebridge-evohome and this works but is limited in functionality. It uses my existing https://www.mytotalconnectcomfort.com/ credentials and so has access to my Evohome.

What I’m struggling with is, how do I connect your suggested https://developer.honeywellhome.com account to my home? I created a new account and made a new app (which coincidentally fails to connect to your plugin), but how will that connect to my home?

Happy New Year!

**Reply by donavanbecker** [reply]
So just to give you the run down. This plugin isn't made for evehome or Total Connect Comfort Thermostats. It is for the Honeywell Home usually T-series thermostats.

There is a potential that you could get it to work by creating a Honeywell home account (which is different then the developer.honeywellhome.com account) and linking it to your Total Connect Comfort account.

![](https://user-images.githubusercontent.com/9875439/103464351-9cba7d80-4cf8-11eb-89bb-770be9a5075a.jpeg)

**Comment by honeywelluser** [comment]
I appreciate the reply. I understand now. Shame the names of the Honeywell products don’t make this more obvious to the uninformed. Evohome ≠ Home 😉 

I got the [other](https://github.com/luc-ass/homebridge-evohome/issues/39#issuecomment-753519082) working with my round thermostat and it works well with a full 12 zone Evohome too.

**Reply by donavanbecker** [reply]
You should try setting it up with the Honeywell home app and see if it works with this plugin. It would be good to know.
